packages/cfd/src/Containers/mt5-compare-table-content.tsx

```diff
@@ -336,7 +336,7 @@ const DMT5CompareModalContent = ({
                 setAppstorePlatform(CFD_PLATFORMS.MT5);
                 setJurisdictionSelectedShortcode('vanuatu');
                 if (
-                    poi_acknowledged_for_vanuatu_maltainvest &&
+                    poi_acknowledged_for_maltainvest &&
```

Comment:
why do we check poi_acknowledged_for_maltainvest for vanuatu

Reply:
Must have missed after renaming the variables. Updated to corresponding flag 👍